configserver/src/main/java/com/yahoo/vespa/config/server/ApplicationRepository.java

```diff
@@ -326,7 +327,9 @@ public ApplicationId activate(Tenant tenant,
         Deployment deployment = deployFromPreparedSession(localSession, tenant, timeoutBudget.timeLeft());
         deployment.setIgnoreSessionStaleFailure(ignoreSessionStaleFailure);
         deployment.activate();
-        return localSession.getApplicationId();
+        ApplicationId applicationId = localSession.getApplicationId();
+        log.log(LogLevel.INFO, "File references used by " + applicationId + ": " + getFileReferences(applicationId));
```

Comment:
The last statement in the prior statement `deployment.activate()` is a log INFO - could we add the file references to that message instead?